Project Mars/MarsQA1/MarsQA-1/Specflow pages/Pages/Language_page.cs

```diff
@@ -0,0 +1,196 @@
+using System;
+using System.Threading;
+using Gherkin.CucumberMessages.Types;
+using MarsQA1.Helper;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using TechTalk.SpecFlow;
+
+namespace MarsQA1.Specflow_Pages;
+
+public class Language_page
```

Comment:
Naming conventions of class is incorrect please fix every where, see this for reference https://www.dofactory.com/csharp-coding-standards